src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -418,15 +419,18 @@ public class LibraryServiceClient implements BackgroundResource {
    * <pre><code>
    * try (LibraryServiceClient libraryServiceClient = LibraryServiceClient.create()) {
    *   ListShelvesRequest request = ListShelvesRequest.newBuilder().build();
-   *   ListShelvesResponse response = libraryServiceClient.listShelves(request);
+   *   for (Shelf element : libraryServiceClient.listShelves(request).iterateAll()) {
+   *     // doThingsWith(element);
+   *   }
    * }
    * </code></pre>
    *
    * @param request The request object containing all of the parameters for the API call.
    * @throws com.google.api.gax.rpc.ApiException if the remote call fails
    */
-  public final ListShelvesResponse listShelves(ListShelvesRequest request) {
-    return listShelvesCallable().call(request);
+  public final ListShelvesPagedResponse listShelves(ListShelvesRequest request) {
```

Comment:
This addition matches https://github.com/googleapis/gapic-generator/blob/eea42da690a5cec5660a76c540a45465323128af/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L568